packages/components/psammead-bulletin/src/index.jsx

```diff
@@ -34,38 +131,74 @@ const IconWrapper = styled.span`
     fill: currentColor;
   }
   margin: 0 0.25rem;
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    display: inline-block;
+    margin-left: 0;
+    padding-left: ${GEL_SPACING};
+  }
 `;
 
 const PlayCTA = styled.div.attrs({ 'aria-hidden': true })`
   background-color: ${({ isLive }) => (isLive ? C_POSTBOX : C_EBON)};
   ${({ service }) => service && getSansRegular(service)};
   ${({ script }) => script && getPica(script)};
   color: ${C_WHITE};
-  text-align: center;
+  align-items: center;
   padding: 0.75rem;
   display: flex;
   justify-content: center;
+  @media (min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MAX}) {
+    display: inline-block;
+    padding: ${GEL_SPACING} ${GEL_SPACING_DBL} ${GEL_SPACING} 0;
+  }
+  @media (min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) {
+    width: 100%;
+  }
 `;
 
 const BulletinSummary = styled.p`
```

Comment:
Is there a reason this isn't separated into a `RadioSummary` and a `TVSummary` in the same way as other components are separated?

Reply:
The difference between the `RadioSummary` and `TvSummary` was quite small. I didn't see it necessary to create separate styled tags.